docs/installation.md

```diff
@@ -21,18 +27,20 @@ We support two installation modes:
     pip install "syntheseus[all]"
     ```
 
-=== "Core (source)"
+=== "Core (GitHub)"
 
     ```bash
+    # NOTE: clone the repository first
     conda env create -f environment.yml
     conda activate syntheseus
 
     pip install -e .
     ```
 
-=== "Full (source)"
+=== "Full (GitHub)"
 
     ```bash
+    # NOTE: clone the repository first
     conda env create -f environment_full.yml
     conda activate syntheseus-full
```

Comment:
Actually, all of the instructions assume one has cloned the repository, as the `pip`-based ones still make use of the conda environments. This is why cloning the repo is mentioned in text below. So, I think we don't need these extra notes inside the instructions.

Reply:
Fair point, I just removed it.